### DISCUSSION TITLE
Any particular reason pgdriver doesn't support transaction isolation levels?

### DISCUSSION THREAD

**Discussion by iamgoroot** [uptrace/bun#1033] [General]
Came across this piece of code
```
	if sql.IsolationLevel(opts.Isolation) != sql.LevelDefault {
		return nil, errors.New("pgdriver: custom IsolationLevel is not supported")
	}
```
Implementation seems pretty straightforward. I'm I missing something? Should I add support for this?

**Comment by vmihailenco** [comment]
I believe this was just not implemented in the first version and none ever asked for it so none bothered.

>Should I add support for this?

Yes, please.

**Comment by iamgoroot** [comment]
Already merged into master
